nodejs/week3/homework/warmup/index.js

```diff
@@ -0,0 +1,67 @@
+const express = require("express");
+const app = express();
+
+app.use(express.urlencoded({extended: true
+}));
+app.use(express.json());
+
+app.get("/", (req, res) => res.send("nodejs week3 homework"));
+
+app.get("/calculator/:method", (req, res) => {
+    if (req.params.method === "add") {
+      let sumResult = addition(req);
+      res.send(sumResult);
+    } else if (req.params.method === "subtract") {
+      const substraction = subtract(req);
+      res.send( substraction);
+    } else if (req.params.method === "multiply") {
+      const multiplication = multiply(req);
+      res.send(multiplication);
+    } else if (req.params.method === "divide") {
+      const division = divide(req);
+      res.send(division);
+    }
+  });
+  
+  app.post("/calculator", (req, res) => {
+     if (req.body.method === "multiply") {
+      const multiplication = multiply(req);
+      res.send(multiplication);
+    } else if (req.body.method === "divide") {
+      const division = divide(req);
+      res.send(division);
+    }});
```

Comment:
Have you tested this with Postman? 

```suggestion
app.post("/calculator/:method", (req, res) => {
  const numbers = Object.values(req.body)
    .flat()
    .map((number) => Number(number));
  if (req.params.method === "multiply") {
    const multiplication = numbers.reduce((acc, value) => acc * value);
    res.send({ multiplication });
  } else if (req.params.method === "divide") {
    const division = numbers.reduce((acc, value) => acc / value);
    res.send({ division });
  }
});
```
☝️ can be a solution 🙂

Reply:
You are still missing Addition and Subtraction methods